src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2121,263 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The definition for subscribing emailId's to query or schedule IDs for a given alertType
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+    properties:
+      assetId:
+        type: string
+        description: The query or schedule ID that the alert is created for and subscribed to.
+      alertType:
+        type: string
+        description: The alert type is one of three values (start, success, failure). This determines the state of query execution that triggers a user to receive an alert.
```

Comment:
Are you testing the render of the swagger? All of the request/response properties are missing `example` attributes. This means that every string property will be given a value of `string`, every boolean `true`, every integer `0`, etc. This does not give the reader an accurate impression of what they should be providing to, or expecting to receive from, the API.

Every property we describe in a swagger needs a `type`, a `description`, and an `example`. The only exception for `example` is for object-type properties that have sub-properties.